HousingRepairsOnlineApi.Tests/GatewaysTests/AppointmentGatewayTests.cs

```diff
@@ -102,16 +103,17 @@ public async Task GivenValidParameters_WhenBookingAppointment_NoExceptionIsThrow
             const string SorCode = "SOR Code";
             const string LocationId = "Location ID";
             const string BookingReference = "Booking Reference";
+            const string RepairDescriptionText = "Repair description text";
             var startDateTime = new DateTime(2022, 01, 01, 8, 0, 0);
             var endDateTime = new DateTime(2022, 01, 01, 12, 0, 0);
-
-            mockHttp.Expect($"/Appointments/BookAppointment?bookingReference={BookingReference}&sorCode={SorCode}&locationId={LocationId}&startDateTime={startDateTime}&endDateTime={endDateTime}")
+            mockHttp.Expect(
+                    $"/Appointments/BookAppointment?bookingReference={BookingReference}&sorCode={SorCode}&locationId={LocationId}&startDateTime={startDateTime}&endDateTime={endDateTime}").With()
```

Comment:
Need to resolve this since it fails a check: There is no argument given that corresponds to the required formal parameter 'matcher' of 'MockedRequest.With(IMockedRequestMatcher)'